src/layout/components/styles/books.css

```diff
@@ -1,49 +1,145 @@
 .book_list_wrapper {
-  padding: 2rem 24px;
   display: flex;
   flex-direction: column;
 }
 
 .book_list_wrapper ul {
   display: flex;
   flex-direction: row;
-  padding-left: 0;
   align-items: center;
-  justify-content: space-between;
-  background-color: rgba(71, 69, 69, 0.334);
+  justify-content: flex-start;
   border-radius: 3px;
 }
 
 .book_list_wrapper ul li {
   list-style: none;
-  padding: 8px 12px;
   font-size: 14px;
 }
 
 .book_list_wrapper button {
   align-self: flex-end;
-  padding: 6px 12px;
 }
 
 @media screen and (min-width: 768px) {
-  .book_list_wrapper {
-    padding: 2rem 4rem;
+  .book_list_display {
     display: flex;
-    flex-direction: column;
     align-items: center;
     justify-content: center;
+    padding: 0.5rem 0;
+  }
+
+  .book_list_wrapper {
+    width: 100%;
+    display: flex;
+    flex-direction: row;
+    align-items: center;
+    justify-content: space-between;
+    background-color: white;
+    border-radius: 5px;
+    padding: 1rem 2rem;
+    box-shadow: rgba(0, 0, 0, 0.05) 0 6px 24px 0, rgba(0, 0, 0, 0.08) 0 0 0 1px;
```

Comment:
- [ ] I noticed that you have multiple repeated CSS properties across your stylesheet. It would be great if you can find a way of reducing this duplication of code ♻️. Remember that not repeating pieces of code will keep your codebase clean and readable. 🤓 🛠️

From [HTML & CSS best practices](https://github.com/microverseinc/curriculum-html-css/blob/main/articles/html_css_best_practices.md):
![image](https://user-images.githubusercontent.com/77069011/200000947-9e58c4d8-fed4-45f8-a822-7828c6e713f7.png)

In this case, you could do this with flex:

```CSS
.flex {
  display: flex;
}
```

And then, use it in your HTML elements where you need a flex container:

```HTML
<div class="flex"> This is a flex container </div>
```

Please, consider fixing this! 🤓